website/docs/r/log_search.html.markdown

```diff
@@ -79,6 +81,15 @@ resource "sumologic_log_search" "example_log_search" {
             threshold_type = "group"
         }
         time_zone = "America/Los_Angeles"
+
+        parameter {
+            name          = "param1"
```

Comment:
Nitpick: 
This is strangely formatted (too much whitespace) and inconsistent with the rest. The same goes for the parameter below.